mmdet/models/dense_heads/rpn_head.py

```diff
@@ -2,16 +2,17 @@
 import torch.nn as nn
 import torch.nn.functional as F
 from mmcv.cnn import normal_init
+from torch.onnx import is_in_onnx_export
 
 from mmdet.ops import batched_nms
-from mmdet.core.post_processing import multiclass_nms
-from mmdet.core.bbox.coder.delta_xywh_bbox_coder import delta2bbox
-from mmdet.core.utils.misc import topk
+from ...core.utils.misc import topk
 from ..builder import HEADS
 from .anchor_head import AnchorHead
 from .rpn_test_mixin import RPNTestMixin
 
 
+
```

Comment:
looks like redundant lines

Reply:
Yes, it has already been removed. Please review the latest version.